Only needs LibUnique 1.1 for GTK_DISABLE_SINGLE_INCLUDES; otherwise 1.0 is fine

--- origsrc/gthumb-2.11.1/configure.ac	2010-01-04 10:33:06.000000000 -0600
+++ src/gthumb-2.11.1/configure.ac	2010-01-25 15:56:01.051008600 -0600
@@ -33,12 +33,12 @@ CLUTTER_REQUIRED=1.0.0
 CLUTTER_GTK_REQUIRED=0.10.0
 GSTREAMER_REQUIRED=0.10
 OPENRAW_REQUIRED=0.0.8
-UNIQUE_REQUIRED=1.1.2
+UNIQUE_REQUIRED=1.0.8
 
 dnl ===========================================================================
 
 CFLAGS=""
-CFLAGS+="-DG_DISABLE_SINGLE_INCLUDES -DGDK_PIXBUF_DISABLE_SINGLE_INCLUDES -DGTK_DISABLE_SINGLE_INCLUDES "
+dnl CFLAGS+="-DG_DISABLE_SINGLE_INCLUDES -DGDK_PIXBUF_DISABLE_SINGLE_INCLUDES -DGTK_DISABLE_SINGLE_INCLUDES "
 CFLAGS+="-DG_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED -DGDK_PIXBUF_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED "
 
 AC_ARG_ENABLE(debug,
